webapp/sources/rudder/rudder-web/src/main/elm/editor/sources/Editor.elm

```diff
@@ -112,8 +112,12 @@ main =
 subscriptions : Model -> Sub Msg
 subscriptions model =
     Sub.batch
-        [ response parseResponse
+        [ draftsResponse parseDraftsResponse
         , updateResources (always (updateResourcesResponse model))
```

Comment:
I'm not sure I understand how resources for a draft are stored. Is it a temp directory derived from the technique id ?

Reply:
Yes exactly ! We create a workspace directory in /var/rudder/configuration-repository that stores the files before the technique is saved, when technique eis savec, resources are moved